<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>636</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QPushButton" name="stoppB">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>70</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>停止</string>
   </property>
  </widget>
  <widget class="QPushButton" name="runpB">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>70</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>运行</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="imageorginGB">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>120</y>
     <width>321</width>
     <height>351</height>
    </rect>
   </property>
   <property name="title">
    <string>原图像</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="imageyoloGB">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>120</y>
     <width>351</width>
     <height>351</height>
    </rect>
   </property>
   <property name="title">
    <string>识别图像</string>
   </property>
  </widget>
  <widget class="QPushButton" name="connectIP">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>490</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>连接</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="ipaddress">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>490</y>
     <width>211</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>490</y>
     <width>69</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>ip地址</string>
   </property>
  </widget>
  <widget class="QPushButton" name="disconnect">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>490</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>断开连接</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
